ignite/internal/tools/gen-mig-diffs/templates/doc/files/{{Version}}.md.plush

```diff
@@ -0,0 +1,17 @@
+---
+sidebar_position: <%= Position %>
+title: <%= ToVersion %>
+description: <%= ShortDescription %>
+---
+
+## **<%= ToVersion %>**
+
+<%= Description %>
+### **Chain migration diffs**
+
+<%= Diffs %>
```

Comment:
Wrapping the diff would work better with the markdown and also would add some syntax highlighting:

````suggestion
```diff
<%= Diffs %>
```
````

Reply:
Maybe a possible future improvement would be to output the diffs grouped in some way that makes sense so after the output is generated it would make it easier for the documentation writer to explain the changes